Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/SpeechInputHandler.cs

```diff
@@ -47,6 +47,7 @@ public SpeechConfirmationTooltip SpeechConfirmationTooltipPrefab
         private readonly Dictionary<string, UnityEvent> responses = new Dictionary<string, UnityEvent>();
 
         #region MonoBehaviour Implementation
+        private bool rememberFocusRequired;
```

Comment:
this seems a bit hacky to me - what magic is happening when manually focusing the speech input handler on start? is there a better way to solve this? and if we have to go with the hack can we comment why we're doing this?